src/main/steps/task-list/content.ts

```diff
@@ -25,6 +26,14 @@ const getSectionStatusLabel = (status, statuses, id) => {
   }
 };
 
+const getAdoptionAgencyUrl = userCase => {
+  if (userCase.adopAgencyOrLAs?.length) {
+    const adopAgency = userCase.adopAgencyOrLAs[0];
+    return `${urls.ADOPTION_AGENCY}?change=${adopAgency.adopAgencyOrLaId}`;
+  }
+  return `${urls.ADOPTION_AGENCY}?add=${Date.now()}`;
+};
```

Comment:
as discussed, please move this method to task-list > utils.ts file
also cover the unit tests for the same